diff --git a/node_modules/duxapp-cli/cjs/lib/util.cjs b/node_modules/duxapp-cli/cjs/lib/util.cjs
index 4b529fb..c2e9e38 100644
--- a/node_modules/duxapp-cli/cjs/lib/util.cjs
+++ b/node_modules/duxapp-cli/cjs/lib/util.cjs
@@ -127,10 +127,9 @@ const randomString = (len = 16) => {
 };
 const asyncSpawn = (cmd, option) => {
   return new Promise((resolve, reject) => {
-    const [_cmd, ..._option] = cmd.split(" ");
-    const task = (0, import_child_process.spawn)(_cmd, _option, {
+    const task = (0, import_child_process.spawn)(cmd, [], {
       stdio: "inherit",
-      shell: process.platform === "win32",
+      shell: true,
       ...option
     });
     task.on("error", (data) => {
diff --git a/node_modules/duxapp-cli/src/index.js b/node_modules/duxapp-cli/src/index.js
index 6de6cf1..411ee80 100755
--- a/node_modules/duxapp-cli/src/index.js
+++ b/node_modules/duxapp-cli/src/index.js
@@ -106,6 +106,9 @@ async function route(route) {
   }
   const lib = (await util.importjs(libPath))
   let func = route[1]
+  if (category === 'ios' && func === 'export') {
+    func = 'exportIpa'
+  }
   if (!lib[func] && lib._index) {
     func = '_index'
   }
diff --git a/node_modules/duxapp-cli/src/lib/ios.js b/node_modules/duxapp-cli/src/lib/ios.js
index c47aad5..9b5ea67 100644
--- a/node_modules/duxapp-cli/src/lib/ios.js
+++ b/node_modules/duxapp-cli/src/lib/ios.js
@@ -29,9 +29,9 @@ export const pod = async (...args) => {
  * @function
  */
 export const build = async () => {
-  const issuerId = getConfig(['ios', 'issuerId'])
-  const keyId = getConfig(['ios', 'keyId'])
-  const keyPath = file.pathJoin(getConfig(['ios', 'keyPath']))
+  const issuerId = await getConfig(['ios', 'issuerId'])
+  const keyId = await getConfig(['ios', 'keyId'])
+  const keyPath = file.pathJoin(await getConfig(['ios', 'keyPath']))
   const execs = [
     'export SKIP_BUNDLING=TRUE && ',
     'xcodebuild -workspace ios/duxapp.xcworkspace -scheme duxapp clean archive -configuration release ',
@@ -49,10 +49,31 @@ export const build = async () => {
  * @function
  */
 export const exportIpa = async () => {
-  const issuerId = getConfig(['ios', 'issuerId'])
-  const keyId = getConfig(['ios', 'keyId'])
-  const keyPath = file.pathJoin(getConfig(['ios', 'keyPath']))
-  const exportOptionsPlist = getConfig(['ios', 'exportOptionsPlist'])
+  const issuerId = await getConfig(['ios', 'issuerId'])
+  const keyId = await getConfig(['ios', 'keyId'])
+  const keyPath = file.pathJoin(await getConfig(['ios', 'keyPath']))
+  let exportOptionsPlist = await getConfig(['ios', 'exportOptionsPlist'], false)
+  if (!exportOptionsPlist) {
+    exportOptionsPlist = file.pathJoin('ios', 'exportOptions.plist')
+    if (!fs.existsSync(exportOptionsPlist)) {
+      const teamID = await getConfig(['ios', 'team'])
+      const method = await getConfig(['ios', 'method'], false) || 'app-store-connect'
+      const content = `<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>method</key>
+	<string>${method}</string>
+	<key>signingStyle</key>
+	<string>automatic</string>
+	<key>teamID</key>
+	<string>${teamID}</string>
+</dict>
+</plist>`
+      exportOptionsPlist = file.pathJoin('dist', 'ios', 'exportOptions.plist')
+      file.writeFile(exportOptionsPlist, content)
+    }
+  }
   const execs = [
     `xcodebuild -exportArchive -archivePath dist/ios/duxapp.xcarchive -exportPath dist/ios/duxapp.ipa -exportOptionsPlist ${exportOptionsPlist} `,
     '-allowProvisioningUpdates ',
@@ -69,8 +90,8 @@ export const exportIpa = async () => {
  * @param filename ipa文件路径
  */
 export const upload = async filename => {
-  const account = getConfig(['ios', 'account'])
-  const password = getConfig(['ios', 'password'])
+  const account = await getConfig(['ios', 'account'])
+  const password = await getConfig(['ios', 'password'])
   await util.asyncSpawn(`xcrun altool --upload-app -t ios -f "${filename}" -u "${account}" -p "${password}"`)
 }
 
diff --git a/node_modules/duxapp-cli/src/lib/util.js b/node_modules/duxapp-cli/src/lib/util.js
index 0ad7958..2762372 100644
--- a/node_modules/duxapp-cli/src/lib/util.js
+++ b/node_modules/duxapp-cli/src/lib/util.js
@@ -92,10 +92,9 @@ export const randomString = (len = 16) => {
 
 export const asyncSpawn = (cmd, option) => {
   return new Promise((resolve, reject) => {
-    const [_cmd, ..._option] = cmd.split(' ')
-    const task = spawn(_cmd, _option, {
+    const task = spawn(cmd, [], {
       stdio: 'inherit',
-      shell: process.platform === 'win32',
+      shell: true,
       ...option
     })
 
